File: AFMobile/features/dev/models/DevUserDTO.ts
// features/dev/models/DevUserDTO.ts
// Speiler DevUserDto i AFBack (Features/Dev/DTOs/DevUserDto.cs).
// Brukes kun av dev-innloggingsskjermen i Development.

export interface DevUserDTO {
  id: string;
  fullName: string;
  email: string;
  profileImageUrl?: string | null;
}

File: AFBack/Features/Dev/DTOs/DevUserDto.cs
using System.ComponentModel.DataAnnotations;

namespace AFBack.Features.Dev.DTOs;

public class DevUserDto
{
    public string Email { get; init; } = null!;
    public string Id { get; set; } = null!;
    public string FullName { get; set; } = null!;
    public string? ProfileImageUrl { get; set; }
}
